module/zfs/vdev_label.c

```diff
@@ -1131,6 +1131,75 @@ vdev_uberblock_load(vdev_t *rvd, uberblock_t *ub, nvlist_t **config)
 }
 
 /*
+ * For use when a leaf vdev is expanded.
+ * The location of labels 2 and 3 changed, and at the new location the
+ * uberblock rings are either empty or contain garbage.  The sync will write
+ * new configs there because the vdev is dirty, but expansion also needs the
+ * uberblock rings copied.  Read them from label 0 which did not move.
+ *
+ * Since the point is to populate labels {2,3} with valid uberblocks,
+ * we silently skip uberblocks we fail to read or which are not valid.
+ */
+
+static int
+vdev_copy_uberblocks(vdev_t *vd)
+{
+	zio_t *zio;
+	int error = 0;
+	int l;
+	int rc;
+	const int src_label = 0;
```

Comment:
Variables which are only used w/in the scope of the `for()` loop below should be declared there.